### PR TITLE
Fixup

### PR DESCRIPTION
**Pull request by donoghuc**

### CODE REVIEW

**Review by mhashizume — APPROVED** [review]

**Comment by donoghuc** [comment]
@mhashizume sorry i pushed one more commit dealing with solaris/sparc issue in 7.x runtime. can you take a look at that commit too?

File: configs/components/rubygem-semantic_puppet.rb [hunk 1]
```diff
@@ -1,11 +1,18 @@
 component "rubygem-semantic_puppet" do |pkg, settings, platform|
   # Projects may define a :rubygem_semantic_puppet_version setting, or we use 1.0.4 by default
   version = settings[:rubygem_semantic_puppet_version] || '1.1.0'
+  if platform.is_cross_compiled? && (platform.is_linux? || platform.is_solaris?)
```
Comment (joshcooper) [inline configs/components/rubygem-semantic_puppet.rb:4]:
```suggestion
  if platform.is_cross_compiled? && platform.is_solaris?
```

Comment (donoghuc) [reply configs/components/rubygem-semantic_puppet.rb]: Updated, thanks!

**Comment by mhashizume** [comment]
> @mhashizume sorry i pushed one more commit dealing with solaris/sparc issue in 7.x runtime. can you take a look at that commit too?

LGTM! 👍

**Pull request merged**